plugins/linear-genome-view/src/LinearGenomeView/components/TrackLabel.tsx

```diff
@@ -30,10 +30,7 @@ const useStyles = makeStyles()(theme => ({
     },
   },
   trackName: {
-    margin: '0 auto',
```

Comment:
from what i could tell, margin and width did nothing so i just removed them here